resources/files/sysv-wrapper.sh

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 
+OLD_LD_LIB=${LD_LIBRARY_PATH//:/ }
```

Comment:
If any of the elements of `LD_LIBRARY_PATH` contain spaces, I think this is gonna be not-great. Maybe awk can help us here?

Supposedly this will also work, but I'm unsure if it's a bash-ism:

```
IFS=':'
OLD_LD_LIB="$LD_LIBRARY_PATH"
for path in $OLD_LD_LIB; do
 ...
done
unset IFS
```

Reply:
"IFS" is "internal field separator"

Reply:
I'm reasonably sure it's a bash-ism.

Reply:
I realized that we have a ruby installed with our package, and ruby sucks way less then bash and CLI.